base/applications/rapps/include/available.h

```diff
@@ -48,17 +64,17 @@ struct CAvailableApplicationInfo
     ATL::CStringW m_szInstalledVersion;
 
     // Create an object from file
-    CAvailableApplicationInfo(const ATL::CStringW& sFileNameParam);
+    CAvailableApplicationInfo(const ATL::CStringW& sFileNameParam, AvailableStrings m_Strings);
 
     // Load all info from the file
-    VOID RefreshAppInfo();
+    VOID RefreshAppInfo(AvailableStrings m_Strings);
```

Comment:
```suggestion
    VOID RefreshAppInfo(AvailableStrings& m_Strings);
```
You probably want to use a reference here, otherwise you create a copy of the object.